src/components/RuntimeManager.js

```diff
@@ -72,7 +73,8 @@ export const RuntimeErrorModal = ({ runtime, onDismiss }) => {
     withErrorReporting('Error loading cloud environment details'),
     Utils.withBusyState(setLoadingRuntimeDetails)
   )(async () => {
-    const { errors: runtimeErrors } = await Ajax().Runtimes.runtime(runtime.googleProject, runtime.runtimeName).details()
+    const { errors: runtimeErrors } = _.lowerCase(runtime.cloudContext.cloudProvider) === cloudProviders.azure.label ? await Ajax().Runtimes.runtimeV2(runtime.workspaceId, runtime.runtimeName).details() :
+      await Ajax().Runtimes.runtime(runtime.googleProject, runtime.runtimeName).details()
```

Comment:
This is how I like to split long tertiaries to call attention to the fact that there is a conditional here:
```suggestion
    const { errors: runtimeErrors } = _.lowerCase(runtime.cloudContext.cloudProvider) === cloudProviders.azure.label
      ? await Ajax().Runtimes.runtimeV2(runtime.workspaceId, runtime.runtimeName).details()
      : await Ajax().Runtimes.runtime(runtime.googleProject, runtime.runtimeName).details()
```